ginivision/src/main/java/net/gini/android/vision/internal/camera/view/FlashButtonHelper.java

```diff
@@ -0,0 +1,34 @@
+package net.gini.android.vision.internal.camera.view;
+
+/**
+ * Created by Alpar Szotyori on 18.02.2019.
+ *
+ * Copyright (c) 2019 Gini GmbH.
+ */
+
+/**
+ * @exclude
+ */
+public final class FlashButtonHelper {
+
+    /**
+     * @exclude
+     */
+    public enum FLASH_BUTTON_POSITION {
```

Comment:
This underscore is kinda weird for the enum name, wouldn't it be better to use `FlashButtonPosition`?

Reply:
Oh, yeah, you're right!